performance/client/src/__tests__/primary/ExplorerAPIs/get-job-output-file-content.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright IBM Corporation 2020
+ */
+
+import WrkTestCase from "../../../testcase/wrk";
+import { getBasicAuthorizationHeader } from "../../../utils";
+
+class ExplorerApiJobOutputFileContentTest extends WrkTestCase {
+  fetchZoweVersions = true;
+
+  name = "Test explorer data sets api endpoint /api/v2/jobs/{jobName}/{jobId}/files/{fileId}/content";
+  endpoint = '/api/v2/jobs/SDSF/STC01060/files/3/content';
```

Comment:
Similar to get-job-details.ts, we will need to find job id dynamically and find "files" list to dynamically update "3".

Reply:
I've modified this test case to first make an API call to retrieve the job ID for the active SDSF job on the system and then use that ID in running the test case. Now am working on getting the file ID dynamically.

Reply:
I created a function in /utils/zowe.ts to get the job output file id dynamically.